src/util/ipc.ts

```diff
@@ -257,6 +257,16 @@ function postSaveConfig(
   }
 }
 
+/**
+ * Post "saveLog" notification.
+ *
+ * Files that take this notification:
+ * - src/main/index
+ */
+function postSaveLog(messages: string): void {
```

Comment:
can we rename the `messages` parameter to `log`? I think the word `messages` implies an array of message while `log` implies one big string that represents the log